### PR TITLE
apps/budgeting: add vote button component and template tag and load v…

### PR DESCRIPTION
**Pull request by phillimorland**
…ia app js

### CODE REVIEW

File: meinberlin/apps/budgeting/assets/react_vote_button_init.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { widget as ReactWidget } from 'adhocracy4'
+import { VoteButton } from './VoteButton.jsx'
+
+function init () {
+  ReactWidget.initialise('mb', 'vote_button',
+    function (el) {
+      const props = el.getAttribute('data-attributes')
+      ReactDOM.render(<VoteButton {...props} />, el)
```
Comment (khamui) [inline meinberlin/apps/budgeting/assets/react_vote_button_init.jsx:10]: somehow the passed props is a string, but should be an object.

File: meinberlin/apps/budgeting/assets/VoteButton.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import django from 'django'
+import CheckboxButton from '../../contrib/assets/CheckboxButton'
+
+export const VoteButton = (props) => {
```
Comment (khamui) [inline meinberlin/apps/budgeting/assets/VoteButton.jsx:5]: I guess you have to wrap it to use as a widget?
Would it make sense to use the `VoteButton` for the list item as well?

**Review by khamui — COMMENTED** [review]
cool! Button is there, just one thing below.

File: meinberlin/apps/budgeting/assets/VoteButton.jsx (hunk 2 above)
Comment (phillimorland) [reply meinberlin/apps/budgeting/assets/VoteButton.jsx:5]: yes it would

File: meinberlin/apps/budgeting/assets/react_vote_button_init.jsx (hunk 1 above)
Comment (phillimorland) [reply meinberlin/apps/budgeting/assets/react_vote_button_init.jsx:10]: ok thats weird, will take a look

Comment (khamui) [reply meinberlin/apps/budgeting/assets/react_vote_button_init.jsx:10]: > somehow the passed props is a string, but should be an object.

turned out, this does not happen.

**Comment by khamui** [comment]
cool! working fine

**Pull request merged**